### PR TITLE
Disable the cAdvisor web-ui.

### PR DESCRIPTION
**Pull request by mvladev**
It should be removed in 1.12.
https://github.com/kubernetes/kubernetes/pull/63881

/cc @ThormaehlenFred

### CODE REVIEW

File: charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet-config-file [hunk 1]
```diff
@@ -14,6 +14,9 @@ authorization:
   webhook:
     cacheAuthorizedTTL: 5m0s
     cacheUnauthorizedTTL: 30s
+{{- if semverCompare "< 1.12" .kubernetes.version }}
+cAdvisorPort: 0
```
Comment (rfranzke) [inline charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet-config-file]: It seems that they haven't ported the `cAdvisorPort` to the `KubeletConfiguration` resource (https://github.com/kubernetes/kubernetes/blob/v1.10.3/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go) as they wanted to remove it anyway in future releases, hence, it's not needed here?

Comment (mvladev) [reply charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet-config-file]: fixed in 01916d9c35129f1ddae0b602c9dab7293c5d9146

File: charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags [hunk 2]
```diff
@@ -13,6 +13,9 @@
 {{- end }}
 --cluster-dns="{{ required "kubernetes.clusterDNS is required" .kubernetes.clusterDNS }}" \
 --cluster-domain={{ required "kubernetes.domain is required" .kubernetes.domain }} \
+{{- if semverCompare "< 1.12" .kubernetes.version }}
+--cadvisor-port=0 \
+{{- end }}
```
Comment (rfranzke) [inline charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags]: Do we want to disable it for all versions currently supported (1.8, 1.9, 1.10)? If so, then we don't need the `if` condition here (see line 2; the whole block is only evaluated for K8S < 1.10).

You also need to add the block above this line here: 
https://github.com/gardener/gardener/blob/master/charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags#L41

Comment (mvladev) [reply charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags]: > Do we want to disable it for all versions currently supported (1.8, 1.9, 1.10)? If so, then we don't need the if condition here (see line 2; the whole block is only evaluated for K8S < 1.10).

The plan is to remove this flag in `1.12`, so it needs not be added for clusters `> 1.12`

Comment (rfranzke) [reply charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags]: OK, thanks, understood. Anyway, the suggested changes need to be applied as described above.

**Review by rfranzke — APPROVED** [review]

**Pull request merged**